core/camel-support/src/main/java/org/apache/camel/support/EndpointHelper.java

```diff
@@ -42,6 +42,8 @@
 
 import static org.apache.camel.util.StringHelper.after;
 
+import java.util.function.Function;
+
```

Comment:
This will fail the codestyle check. Try to run the build with the profile sourcecheck enabled and fix the warnings reported. Thanks.

Reply:
Thanks, will do